app/templates/ember-community-survey-2019.hbs

```diff
@@ -99,7 +99,7 @@
     <a href="#SS_Q104" id="SS_Q104" class="text-lg">
       How long have you been working with Ember?
     </a>
-    <HighCharts @chartOptions={{SS_Q104.options}} @content={{SS_Q104.data}} @theme={{theme}} />
+    <HighCharts @chartOptions={{SS_Q104.options}} @content={{SS_Q104.data}} @theme={{this.theme}} />
```

Comment:
**suggestion (non-blocking):**

In lines 102, 175, and 258, we can add `this.`. (Not crucial since the code is commented out, but nice to do for consistency.)

```suggestion
    <HighCharts @chartOptions={{this.SS_Q104.options}} @content={{this.SS_Q104.data}} @theme={{this.theme}} />
```